### ISSUE TITLE
Adding a custom Time Range Filter inside the Dashboard

### ISSUE THREAD

**Issue by Gill-Bates** [louislam/uptime-kuma#4916]
### 📑 I have found these related issues/pull requests

x

### 🏷️ Feature Request Type

Dashboard

### 🔖 Feature description

I would like to define a custom time range to display the chart. The Maximum is 1 week:

![chrome_XRxHgQ33Lb](https://github.com/louislam/uptime-kuma/assets/25774289/001a0687-f526-4205-878d-09c69a94ddd9)

### ✔️ Solution

Extend the menu with a Date/Time Picker (Start... End...)

**Comment by CommanderStorm** [comment]
Duplicate of https://github.com/louislam/uptime-kuma/issues/2477

TL;DR: Currently blocked on benchmarking if the performance in v2 is good enough to allow this

**Comment by CommanderStorm** [comment]
(I should have closed this as a duplicate)

**Comment by gudge25** [comment]
Same for me 
if I set every 90 seconds
My time range on a dashboard is only 45 min, I hope it can be at least one week

<img width="591" alt="Image" src="https://github.com/user-attachments/assets/ee009796-00d1-4eae-96e9-b98be46b4080" />
